src/Uno.UI/UI/Xaml/Media/Animation/DoubleAnimation.cs

```diff
@@ -342,7 +342,7 @@ private void OnEnd()
 
 			if (FillBehavior == FillBehavior.HoldEnd)//Two types of fill behaviors : HoldEnd - Keep displaying the last frame
 			{
-#if __IOS__
+#if __IOS__ || __MACOS__
 				// iOS: Here we make sure that the final frame is applied properly (it may have been skipped by animator)
```

Comment:
You should adjust the comments to include macOS too when appropriate.

Reply:
Fixed